pages/real-weddings/wedding.jade

```diff
@@ -47,8 +47,8 @@ block main
         h2.h1.text-xs-center.text-uppercase.text-primary.m-b-2 - Suppliers - 
         .row
           .col-sm-10.col-md-8.col-sm-offset-1.col-md-offset-2
-            each supplier in content._entry.suppliers
-              p.text-xs-center=supplier
+            p.text-xs-center
+              !=md.renderInline(content._entry.suppliers)
```

Comment:
Why `renderInline`? If Freya adds a UL it might not render properly...

How about:

```
.text-xs-center
  !=md.render(content._entry.suppliers)
```